wodles/aws/aws-s3.py

```diff
@@ -718,22 +864,253 @@ def load_information_from_file(self, log_key):
             tsv_file = csv.DictReader(f, fieldnames=fieldnames, delimiter=' ')
             return [dict(x, source='vpc') for x in tsv_file]
 
+    def get_ec2_client(self, access_key, secret_key):
+       conn_args = {}
+       if access_key is not None and secret_key is not None:
+           conn_args['aws_access_key_id'] = access_key
+           conn_args['aws_secret_access_key'] = secret_key
+
+       boto_session = boto3.Session(**conn_args)
+
+       try:
+           ec2_client = boto_session.client(service_name='ec2')
+       except Exception as e:
+           print("Error getting EC2 client: {}".format(e))
+           sys.exit(3)
+
+       return ec2_client
+
+    def get_flow_logs_ids(self, access_key, secret_key):
+        ec2_client = self.get_ec2_client(access_key, secret_key)
+        flow_logs = ec2_client.describe_flow_logs()['FlowLogs']
+        flow_logs_ids = []
```

Comment:
Appending to lists can be very slow. Use a one-liner to build the list:
```suggestion
        flow_logs_ids = list(map(operator.itemgetter('FlowLogId'), ec2_client.describe_flow_logs()['FlowLogs']))
```